### PR TITLE
Skitch octocat-json.png

### PR DESCRIPTION
**Pull request by ShayHall**

### CODE REVIEW

**Comment by jmcreasman** [comment]
About as flawless as it could be! Great job!

**Comment by Ricecoder** [comment]
AMAZING! There is nothing I can say. Good job :+1:

File: src/img/octicons/octicons.css [hunk 1]
```diff
@@ -0,0 +1,221 @@
+@font-face {
```
Comment (kellymurray) [inline src/img/octicons/octicons.css:1]: Consider linking the CDN in `index.html` instead of installing all the font files!

File: src/index.html [hunk 2]
```diff
@@ -9,7 +9,7 @@
 
         <link rel="apple-touch-icon" href="apple-touch-icon.png">
         <!-- Place favicon.ico in the root directory -->
-
+        <link rel="stylesheet" href="img/octicons/octicons.css">
```
Comment (kellymurray) [inline src/index.html:12]: CDN time!

**Pull request merged**